index.html

```diff
@@ -66,8 +66,11 @@
   </main>
   <script src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/handlebars.js/4.0.5/handlebars.js"></script>
-  <script type="text/javascript" src="app/appData.js"></script>
+  <!-- <script type="text/javascript" src="app/appData.js"></script> -->
```

Comment:
You can just delete this line, instead of commenting it out. It will still appear in prior versions of this file in GitHub if you need to find it again.